src/index.html

```diff
@@ -10,23 +10,39 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <section>
+      <form
+        class="search"
+        data-qa="big"
+      >
+        <input
+          class="search__input search__input--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
+    </section>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <section>
+      <form
+        class="search"
+        data-qa="small"
+      >
+        <input
+          class="search__input search__input--small"
+          type="text"
+          placeholder="Try “Los Angeles“"
```

Comment:
The `data-qa` attribute for the small search form input is missing. According to the previous review, this attribute should be removed as it doesn't meet the task requirements.